proca/config/config.exs

```diff
@@ -55,6 +55,11 @@ config :proca, Proca,
   start_daemon_servers: true,
   mtt_cycle_time: 3
 
+config :proca, Proca.Service.EmailBackend,
+  srs_key: System.get_env("EMAIL_SRS_KEY") || "teiy1sah8seengiem0ee2Yai",
```

Comment:
I'm not a fan of hardcoded default keys, if EMAIL_SRS_KEY isn't set can you generate a random key and print it in the log "WARNING: MISSING env variable EMAIL_SRS_KEY, using " + generated key

Reply:
oh you mean generate during config time? ehh, right, I thought it could be problematic but actually the whole thing is useless without a set key so it's fine

it does warn if missing, currently at runtime but maybe it should do so at config time

Reply:
aah, my main issue is to prevent putting a hardcoded key by default that could be easily guessed by an attacker.

In that case, that would allow the attacker to verify that the SRS is valid... so not the most very important one, that key could be public anyway. I guess it's more to prevent a pattern in general than for that specific case ;)